styles/_myft-card.scss

```diff
@@ -70,3 +73,10 @@
 		color: getColor('black-80');
 	}
 }
+
+.topic-card__myft-footer__divider {
+	height: 18px;
+	width: 1px;
+	background: getColor('black-30');
```

Comment:
I wonder if this should have the same border colour as the other dividers `black-10`?